NEWS.md

```diff
@@ -67,6 +67,9 @@
 
 * `rescale_mid()` now properly handles NAs (@foo-bar-baz-qux, #104).
 
+* `train_continuous()` now maintains the class of inputs when they are not numeric
```

Comment:
@hadley is this going to make the release? I suspect it won't and this should actually be in its own new section up top: 
```
# scales 1.0.0.9000

*  ...
```

Reply:
When this is resolved, I'll provide a new PR.

Reply:
So no, this change will not be included in the 1.0.0 release. Please move this news bullet to the top with the new header I demonstrated above, and add your github handle and the issue number so that it follows [style guidelines](http://style.tidyverse.org/news.html#news-bullets). No need to open a new PR, just make a new commit to your branch with the changes.